packages/core/src/tracing/instrumentation/protocols/httpClient.js

```diff
@@ -186,12 +186,7 @@ function instrument(coreModule, forceHttps) {
     // packages/core/src/tracing/clsHooked/unset.js#storeReducedSpan.
     const parentSpan = cls.getCurrentSpan() || cls.getReducedSpan();
 
-    if (
-      skipTracingResult.skip ||
-      !parentSpan ||
-      constants.isExitSpan(parentSpan) ||
-      shouldBeBypassed(parentSpan, options)
-    ) {
+    if (skipTracingResult.skip || shouldBeBypassed(parentSpan, options)) {
```

Comment:
```
const resp = cls.skipExitTracing({
  checkReducedSpan: true,
  extendedResponse: true,
  skipIsTracing: true
})
...
if (skipTracingResult.skip || shouldBeBypassed(resp.parentSpan, options)) {
```

Reply:
```shouldBeBypassed(resp.parentSpan, options)``` this check is specific to httpClient alone. IMO we can keep this in here itself. no need to move to cls